readthedocs/subscriptions/products.py

```diff
@@ -101,20 +122,31 @@ def get_feature(obj, feature_type) -> RTDProductFeature:
         else:
             raise TypeError
 
+        # This happens when running tests on .com only.
+        # In production projects are always associated with an organization.
+        if not organization:
+            return settings.RTD_DEFAULT_FEATURES.get(feature_type)
+
         # A subscription can have multiple products, but we only want
-        # the product from the organization that has the feature we are looking for.
-        stripe_products_id = [
+        # the products from the organization that has the feature we are looking for.
+        available_stripe_products_id = [
             product.stripe_id for product in get_products_with_feature(feature_type)
         ]
-        stripe_product_id = (
-            djstripe.Product.objects.filter(
-                id__in=stripe_products_id,
-                prices__subscription_items__subscription__rtd_organization=organization,
-            ).values_list("id", flat=True)
-            # TODO: maybe we should merge features from multiple products?
-            .first()
-        )
-        if stripe_product_id is not None:
-            return settings.RTD_PRODUCTS[stripe_product_id].features[feature_type]
+        stripe_subscription = organization.get_or_create_stripe_subscription()
+        if stripe_subscription:
+            subscription_items = stripe_subscription.items.filter(
+                price__product__id__in=available_stripe_products_id
+            ).select_related("price__product")
+            final_rtd_feature = None
+            for subscription_item in subscription_items:
+                rtd_feature = settings.RTD_PRODUCTS[
+                    subscription_item.price.product.id
+                ].features[feature_type]
+                if final_rtd_feature is None:
```

Comment:
Won't this always be None since we're setting it to None a couple lines above?

Reply:
It'll be none just on the first iteration, on consecutive iterations it'll be set.

Reply:
Ah gotcha.